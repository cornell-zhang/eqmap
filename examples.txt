// MUX with exact inputs
(MUX s1 (MUX s0 a b) (MUX s0 a b)) // Simplifies to a (LUT 202 s0 a b)

// MUX with match cuts
(MUX s1 (MUX s0 a b) (MUX s0 b a)) // Simplifies to a (LUT 44234 s1 s0 b a)

// 4:1 mux
(MUX s1 (MUX s0 a b) (MUX s0 c d)) // Simplifies to a (LUT 18374951396690406058 s1 s0 a b c d)
