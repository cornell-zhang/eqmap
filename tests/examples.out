(LUT 202 s0 a b)
(LUT 44234 s1 s0 b a)
(LUT 51952 s1 (LUT 61642 s1 s0 c d) a b)
(LUT 202 s0 a b)
